trim.production.js

```diff
@@ -144,16 +258,27 @@ module.exports = function (apiUrl, apiToken, debug) {
   assert(typeof apiUrl === 'string', 'Argument 1 to instantiate the TRIM wrapper must be a valid url.')
   assert(typeof apiToken === 'string', 'Argument 2 to instantiate the TRIM wrapper must be a valid apiToken.')
 
-  url = apiUrl;
-  token = apiToken;
+  url = apiUrl
+  token = apiToken
   if (!!debug)
-    require('request-debug')(request);
+    require('request-debug')(request)
 
   return {
```

Comment:
`getPrivateContainer` at least was part of the previous api. My original intention was that we would just expose the public/private versions of the method and the raw one would be private. I think you're right though, that the base version of the methods are straightforward enough now that they accept objects instead of a long string of parameters that we can just kill the public/private versions.

I would have to keep the getPrivateContainer method and add a deprecation warning.